app/workers/rapid_ready_for_decision/form526_asthma_job.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'lighthouse/veterans_health/client'
+
+module RapidReadyForDecision
+  class Form526AsthmaJob < Form526BaseJob
+    STATSD_KEY_PREFIX = 'worker.fast_track.form526_asthma_job'
+
+    sidekiq_options retry: 2
+
+    def assess_data(form526_submission)
+      client = lighthouse_client(form526_submission)
+      client.list_resource('medication_requests')
```

Comment:
Do we want to include some aspect of `medication_requests` result in the body of the message? If not, why are we making a query here?

Reply:
Good call. I was going to just let it make the roundtrip to Lighthouse, but I'll update this to at least report back the number of returned resources.